src/sharding/ShardingManager.js

```diff
@@ -43,6 +43,12 @@ class ShardingManager extends EventEmitter {
     const stats = fs.statSync(this.file);
     if (!stats.isFile()) throw new Error('CLIENT_INVALID_OPTION', 'File', 'a file');
 
+    /**
+     * Enables the ability to rebalance the Shards.
+     * @type {boolean}
+     */
+    this.autoShards = this.totalShards === 'auto';
```

Comment:
This will always be false because `this.totalShards` has not been defined yet.

Reply:
wow.. well.. I feel dumb. I'll look into it.

Reply:
yah cant believe i did that. :|